src/pages/academy/components/_search-banner.tsx

```diff
@@ -377,7 +378,20 @@ const SearchBar = ({ setModal, setHideMobileTopic }: SearchBarProps) => {
 
     const input_ref = useRef<HTMLInputElement>()
 
-    const combined_data = [...academy_data.blog, ...academy_data.videos]
+    let filtered_academy_data = academy_data.blog
+
+    if (is_eu) {
+        filtered_academy_data = academy_data.blog.filter(
+            (item) => item.visibility !== 'hide_for_eu' && item.visibility !== 'hide_for_eu_uk',
+        )
+    } else if (is_uk) {
+        filtered_academy_data = academy_data.blog.filter(
+            (item) => item.visibility !== 'hide_for_uk' && item.visibility !== 'hide_for_eu_uk',
+        )
+    }
+
+    const combined_data = [...filtered_academy_data, ...academy_data.videos]
```

Comment:
Could geolocation search in case of videos be implemented ?

Reply:
Oh thanks I only now noticed that we have the same property to query as in articles